pkg/apis/serving/v1beta1/route_validation.go

```diff
@@ -75,20 +75,20 @@ func (rs *RouteSpec) Validate(ctx context.Context) *apis.FieldError {
 
 // Validate verifies that TrafficTarget is properly configured.
 func (tt *TrafficTarget) Validate(ctx context.Context) *apis.FieldError {
-	var errs *apis.FieldError
+	errs := tt.validateLatestRevision(ctx)
+
+	errs = tt.validateRevisionAndConfiguration(ctx, errs)
+
+	errs = tt.validateTrafficPercentage(errs)
 
+	errs = tt.validateUrl(ctx, errs)
```

Comment:
the last one can be simply returned. And probably empty lines, are a bit redundant

Reply:
Done